app/templates/_package.json

```diff
@@ -91,7 +91,8 @@
         "karma-jasmine": "~0.2.0",<% } else if (testFramework === 'mocha') { %>
         "karma-chai": "0.1.0",
         "karma-mocha": "~0.1.3",<% } %>
-        "karma": "~0.12.21",<% } %>
+        "karma": "~0.12.21",<% } %><% if (jsFramework != 'react') { %>
```

Comment:
Can you change this to `!==`? I'd like to avoid type conversion when it is not needed.